### ISSUE TITLE
[App Support]: I hope PlayCover supports mobile touch screen operation and achieves cloud gaming like effects

### ISSUE THREAD

**Issue by swordman20** [PlayCover/PlayCover#1532]
### App Store link

希望PlayCover支持手机触屏操作，能够把运行的游戏，投屏到手机，手机触屏能够控制，实现云游戏一样的功能，把游戏运行放在Mac上，操作指令由手机多点控制。

### Current issues

- [ ] App fails to install
- [ ] App fails to open
- [ ] App crashes randomly during usage
- [ ] App experiences graphical issues or broken UI
- [ ] App fails to connect to servers
- [ ] Other

### More details

I hope PlayCover can support touch screen operation on mobile phones, and can project running games onto the phone. The phone touch screen can be controlled, achieving the same function as cloud games. The game can be run on a Mac, and the operation instructions can be controlled by multiple points on the phone.

### What version of PlayCover are you using?

3.0.0 beta 2

### What version of macOS are you using?

Sonoma (macOS 14)

### Issue Language

- [X] Yes my issue is written in English

**Comment by TheMoonThatRises** [comment]
I'm not sure I understand the request. PlayCover does support touch-screen like operations, using the mouse and keymapping.

**Comment by mrjasonzee** [comment]
> I'm not sure I understand the request. PlayCover does support touch-screen like operations, using the mouse and keymapping.

i think what they mean is streaming the game running on the mac to an iphone so they can use a touchscreen. think of reverse sidecar

**Comment by swordman20** [comment]
> > I'm not sure I understand the request. PlayCover does support touch-screen like operations, using the mouse and keymapping.
> 
> i think what they mean is streaming the game running on the mac to an iphone so they can use a touchscreen. think of reverse sidecar

Yes, using the powerful processor of Mac, running large mobile games, passing them on to iPhone, and being able to control the games running on Mac on iPhone
